extension/src/paymentHandler/refund-payment.handler.js

```diff
@@ -0,0 +1,39 @@
+const Promise = require('bluebird')
+const pU = require('./payment-utils')
+const { refund } = require('../service/web-component-service')
+const {
+  CTP_INTERACTION_TYPE_REFUND,
+} = require('../config/constants')
+
+async function execute (paymentObject) {
+  const refundInitTransactions = pU.listRefundTransactionsInit(paymentObject)
+  const chargeSuccessTransaction = pU.getChargeTransactionSuccess(paymentObject)
+
+  const actions = []
```

Comment:
```suggestion
  const updateActions = []
```

Reply:
We name it actions in other handlers as well. It's not bad name, so I would keep it.